packages/core/README.md

```diff
@@ -21,11 +21,14 @@ The agent requires a configuration parameter.
 
 | Property                | Description                                                             | Type                | Optional | Default Value            |
 | ----------------------- | ----------------------------------------------------------------------- | ------------------- | -------- | ------------------------ |
-| `globalObjectKey`       | String that should be used when defining the agent on the global object | `string`            | Y        | `grafanaJavaScriptAgent` |
-| `metas`                 | Array of metas that should be logged                                    | `Meta[]`            | Y        | `[]`                     |
+| `app`                   | Application metadata                                                    | `App`               | Y        | `undefined`              |
+| `globalObjectKey`       | String that should be used when defining the agent on the global object | `string`            | Y        | `grafanaAgent`           |
+| `metas`                 | Array of metas that should be logged                                    | `MetaItem[]`        | Y        | `[]`                     |
 | `instrumentations`      | Array of instrumentations that should be ran                            | `Instrumentation[]` | N        | `[]`                     |
 | `preventGlobalExposure` | Flag for toggling the definition on the global object                   | `boolean`           | Y        | `false`                  |
 | `transports`            | Array of transports that should be used                                 | `Transport[]`       | Y        | `[]`                     |
+| `session`               | Session metadata                                                        | `Session`           | Y        | `undefined`              |
```

Comment:
`preventGlobalExposure` > `session` > `transports`

Reply:
I need a linter for this, my brain does not include a sort function :D

Reply:
It's called OCD.  Some may consider it a bug, but in this case it is a feature.